packages/imperative/__tests__/config/cmd/secure/secure.handler.test.ts

```diff
@@ -100,8 +100,7 @@ describe("Configuration Secure command handler", () => {
                 }),
                 save: ((k: string, v: any): Promise<void> => {
                     return CredentialManagerFactory.manager.save(k, v);
-                }),
-                name: CredentialManagerFactory.manager.name
+                })
```

Comment:
Seems like the vault.name was removed a little over 7 months ago from the IConfigVault interface.
Interesting how this test compiled/transpile 😋